packages/commonwealth/client/scripts/views/components/user/user.tsx

```diff
@@ -204,9 +204,9 @@ export const User = ({
               </Link>
             )}
           </div>
-          {profile?.address && (
-            <div className="user-address">{redactedAddress}</div>
-          )}
+          <div className="user-address">
+            {profile?.address ? redactedAddress : 'Address removed'}
+          </div>
```

Comment:
Is this the expected look? It says "Address removed," but the truncated address is still visible where the Profile name would be.

<img width="356" alt="Screenshot 2023-11-08 at 12 06 13 PM" src="https://github.com/hicommonwealth/commonwealth/assets/9438198/248e6aae-0682-4d6f-b8e1-e3896f703ada">